### PR TITLE
Add moltemplate tutorial to HowTo section

### PR DESCRIPTION
**Pull request by akohlmey**
**Summary**

This adds a brief tutorial for setting up LAMMPS inputs with Moltemplate based on discussions from https://matsci.org/t/opls-aa-force-field/51987

**Author(s)**

Otello M Roscioni, MaterialX Ltd,  om.roscioni@materialx.co.uk 

submitted by @akohlmey 

**Licensing**

By submitting this pull request, I agree, that my contribution will be included in LAMMPS and redistributed under either the GNU General Public License version 2 (GPL v2) or the GNU Lesser General Public License version 2.1 (LGPL v2.1).

**Backward Compatibility**

N/A

**Implementation Notes**

Copies of the quoted files are provided in "tools/moltemplate/tutorial-files"

**Post Submission Checklist**

- [x] The feature or features in this pull request is complete
- [x ] Licensing information is complete
- [x] Corresponding author information is complete
- [x] Suitable new documentation files and/or updates to the existing docs are included
- [x] The added/updated documentation is integrated and tested with the documentation build system

### CODE REVIEW

**Review by stanmoore1 — APPROVED** [review]
Nice work--I approve.

File: doc/src/Howto_moltemplate.rst
Comment (sjplimp) [inline doc/src/Howto_moltemplate.rst:5]: ... use the Moltemplate tool, included in the LAMMPS distribution, to setup a classical ...

**Review by sjplimp — CHANGES_REQUESTED** [review]
Thanks for creating this tutorial!  I made a small comment for the first paragraph of the Howto file.

I also think we should wait for a review from Andrew before merging this, as he
may have more detailed comments.

File: doc/src/Howto_moltemplate.rst
Comment (akohlmey) [reply doc/src/Howto_moltemplate.rst:5]: Moltemplate hasn't been included in the LAMMPS distributions since January 2019. There is only a README file now explaining how to install it.

Comment (sjplimp) [reply doc/src/Howto_moltemplate.rst:5]: oops - I didn't read the Tools section carefully enough.  You can ignore my suggested Howto doc change

**Review by sjplimp — APPROVED** [review]

**Pull request merged**
